modules/devkit-support/src/main/java/org/mule/security/oauth/processor/BaseOAuth2AuthorizeMessageProcessor.java

```diff
@@ -134,12 +136,16 @@ private void setState(Map<String, String> extraParameters, MuleEvent event)
 
                 try
                 {
+                    JsonProperty property=field.getAnnotation(JsonProperty.class);
```

Comment:
We cannot use an alias on DevKit because this is using reflection to add the field from the class.

I'll add a new annotation to decouple if from Jackson. 

Using an annotation was one possible way of avoiding modifying org.mule.common.security.oauth.AuthorizationParameter class to add an alias/propertyName field